src/OpenAL/Silk.NET.OpenAL/ALC/ALContext.cs

```diff
@@ -110,12 +111,26 @@ public static unsafe ALContext GetApi(bool soft = false)
             return ret;
         }
 
+        /// <summary>
+        /// Attempts to load the given extension.
+        /// </summary>
+        /// <typeparam name="T">The extension type.</typeparam>
+        /// <param name="device">The device the context is on.</param>
+        /// <param name="ext">The extension to check for.</param>
+        /// <returns>Whether the extension is available.</returns>
+        public unsafe bool TryGetExtension<T>
+            (Device* device, out T ext) where T : NativeExtension<ALContext> => 
+            !((ext = IsExtensionPresent(device, ExtensionAttribute.GetExtensionAttribute(typeof(T)).Name)
+                ? (T) Activator.CreateInstance(typeof(T), Context)
+                : null) is null);
+
         /// <summary>
         /// Gets an instance of the API of an extension to the API.
         /// </summary>
         /// <typeparam name="TContextExtension">The extension type.</typeparam>
         /// <param name="device">The device the context is on.</param>
         /// <returns>The extension.</returns>
+        [Obsolete("This method has been deprecated and will be removed in Silk.NET 3.0. Please use TryGetExtension instead.")]
```

Comment:
Same again, apologies, my bad!

```suggestion
        [Obsolete
        (
            "This method has been deprecated and will be removed in Silk.NET 3.0. " +
            "Please use TryGetExtension instead."
        )]
```